docs/docker-hub/index.md

```diff
@@ -16,23 +16,23 @@ management, user and team collaboration, and lifecycle workflow automation.
 
 ![DockerHub](/docker-hub/hub-images/hub.png)
 
-## [Finding and pulling images](./userguide.md)
+## [Finding and pulling images](userguide.md)
```

Comment:
+1 Headings should be headings not links; especially because these headings also contain links in the body text.